test/descriptive-link-text-override-config.md

```diff
@@ -0,0 +1,17 @@
+# Descriptive link text override config
```

Comment:
I'm getting nitpicks now, but please remove "config" from the end - it's just an override. (Sorry!)

Reply:
I may do a final nitpick pass on your branch to make some of these silly changes myself instead of annoying you. :)